ingest-app/project.clj

```diff
@@ -12,6 +12,7 @@
                  [commons-codec/commons-codec "1.11"]
                  [commons-io "2.6"]
                  [compojure "1.6.1"]
+                 [digest "1.4.8"]
```

Comment:
I think we should just define the md5 function in `cmr.common.util` rather than bringing in another dependency.

```
(defn md5
  "Return the md5 for a string. See https://gist.github.com/jizhang/4325757."
  [^String s]
  (->> s
       .getBytes
       (.digest (MessageDigest/getInstance "MD5"))
       (BigInteger. 1)
       (format "%032x")))
```

Reply:
I don't see the benefit of putting one off functions in `cmr.common.util`. The `clj-digest` library is a cleaner way to provide that functionality. Also, I have thought about using some digest other than md5 (e.g. Sha1 or sha-256 as they suffer less collision) and that is why I have made the fingerprint column in CMR_VARIABLES a VARCHAR(64) rather than VARCHAR(32). Using the `clj-digest` library will make the switch easy if desired.

Reply:
Alright if no vulnerabilities are found I won't fight you (`lein check-sec` runs successfully).

Reply:
Thanks! if anything, I think `clj-digest` would be more secure than rolling our own, and it is such a simple small wrapper lib, we are not pulling in much.